### PR TITLE
Feature/sdl 0269 new vehicle data climate data

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Fixes https://adc.luxoft.com/jira/browse/FORDTCN-7856

https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0269-New-vehicle-data-ClimateData.md

This PR is **[ready]** for review.

### Summary
Add new vehicle data ClimateData

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,7 +274,8 @@ Error code, which comes from the module side.
 |`APP_UNAUTHORIZED`||
 |`PROTOCOL_VIOLATION`||
 |`UNSUPPORTED_HMI_RESOURCE`||
-|`RESOURCE_CONSTRAINT`||
+|`RESOURCE_CONSTRAINT`|By sending this value, SDL unregisters the application to allow the HMI to close the application.|
```
Comment (AKalinich-Luxoft) [inline README.md]: @LitvinenkoIra  I believe this is not related to proposal

**Review by ychernysheva — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LitvinenkoIra) [reply README.md]: @AKalinich-Luxoft Agree, removed. Please check the latest updates

**Review by AKalinich-Luxoft — APPROVED** [review]

**Review by VladSemenyuk — APPROVED** [review]

**Comment by LitvinenkoIra** [comment]
Internal review is done. Move PR to the open
